index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">
+          <img src="images/Union.png" alt="menubar" />
+        </a>
+      </nav>
+    </header>
+    <main>
+      <section class="Port">
+        <div hreh="#" class="Menu"></div>
+        <h1>
+          Hey there!<br />
+          I'm Michael
+        </h1>
+
+        <h2>
+          I'm a Software<br />
+          Developer
+        </h2>
+
+        <p>
+          I can help you build a product,<br />
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </p>
+
+        <ul id="SocMediaHand">
+          <li>
+            <a href="https://www.linkedin.com/in/lawrence-amoafo-90b57213a/"
+              ><ion-icon name="logo-linkedin"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="https://github.com/Mike47ip"
+              ><ion-icon name="logo-github"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="https://twitter.com/Mike_pee47"
+              ><ion-icon name="logo-twitter"></ion-icon>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <ion-icon name="logo-facebook"></ion-icon>
+            </a>
+          </li>
+        </ul>
+
+        <div class="downarrow">
+          <img src="images/Disabled.png" alt="rockbuttom arrow" />
+        </div>
+      </section>
```

Comment:
Your design
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/78251927/d52a711b-377a-4150-a9eb-fefa7ea277a5)

Figma
![image](https://github.com/Mike47ip/mobilePortfolioWeb/assets/78251927/cafbf9ee-d6ed-40cf-9f44-d462258d5ec5)


To make our website look exactly like the `Figma design`, kindly remove the margin `right` and `left` and ensure that the background images cover the entire width of the section. 🛠💪

* If it's pretty and works it will be a wonderful website. 🤗